app/settings/data-export/hdx-export.controller.js

```diff
@@ -116,37 +120,63 @@ function (
     }
 
     function formatIds() {
-        let data = [];
+        let hxlData = [];
         _.each($scope.forms, (form) => {
             _.each(form.attributes, (formAttribute) => {
                 if (formAttribute.selected && formAttribute.selected.length > 0) {
-                    let ids = {
-                            form_attribute_id : formAttribute.id,
-                            hxl_tag: null
-                        };
-
-                    if (formAttribute.selectedTag) {
-                        ids.hxl_tag = {
-                            hxl_tag_id: formAttribute.selectedTag.id
-                        };
-
-                        if (formAttribute.selectedHxlAttributes) {
-                            ids.hxl_tag.hxl_attribute_ids = [];
-                            _.each(formAttribute.selectedHxlAttributes, (hxlAttribute) => {
-                                ids.hxl_tag.hxl_attribute_ids.push(hxlAttribute.id);
-                            });
-                        }
+                    let obj = formAttribute.selectedTag ? {form_attribute_id: formAttribute.id, hxl_tag_id: formAttribute.selectedTag.id} : {form_attribute_id: formAttribute.id};
```

Comment:
I think it might be cleaner to just create the obj in the if/else, since this adds some unnecessary complexity.
 
(mandatory "ignore if you feel this is nitpicky and dumb" comment here)